esphome/components/opentherm/generate.py

```diff
@@ -0,0 +1,140 @@
+from collections.abc import Awaitable
+from typing import Any, Callable
+
+import esphome.codegen as cg
+from esphome.const import CONF_ID
+from . import const
+from .schema import TSchema
+
+opentherm_ns = cg.esphome_ns.namespace("esphome::opentherm")
```

Comment:
Shouldnt need this part
```suggestion
opentherm_ns = cg.esphome_ns.namespace("opentherm")
```